doc/nginx-setup.md

```diff
@@ -18,7 +18,7 @@ block
 
```

Comment:
Not overly keen on doing a script because:
  - It's running with sudo
  - It's messing with you global config. If it's messed up in some way, hard to know what it touches with going deep into the code

Also, it's a 5 minutes one-off step, which is negligible if you're going to do any work on the project.